OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -75,60 +96,72 @@ public sealed class DreamObjectWorld : DreamObject {
             _viewRange = new ViewRange(viewInt);
         }
     }
-
```

Comment:
Put back the empty lines around `HandleDeletion()`, as well as above `OperatorOutput()`